tools/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
+
+## 2025 February
+
+### Changed
+
+- `ngsderive.encoding` removed the `String inferred_encoding` output [#216](https://github.com/stjudecloud/workflows/pull/216).
```

Comment:
@adthrasher I considered adding an entry for the `global_phred_scores` task, but I actually don't think that is a change of substance for users. The task API isn't changed, so I don't think users should care if the script is in-lined or not.

Reply:
That's true, but the script is in some way a new feature now that it is standalone.